### PR TITLE
Single video and history

### PR DESCRIPTION
**Pull request by akash-1318**
## Implemented single video page and History - 
I have implemented single video page and history in my video library.

## Files to check - 
1. src/pages/history/history.js
2. src/pages/single-video.js/single-video.js
3. src/utils/handle-history.j

https://user-images.githubusercontent.com/64393554/167954454-5edcb83a-9d5f-48e8-a11a-5f114d1af40f.mov

s

### CODE REVIEW

**Review by srv-pranjal — APPROVED** [review]
Good Work.
Clean Code, easy to understand.
UI looks great.

File: src/App.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ import {Signup} from "./pages/index"
 import {Login} from "./pages/index"
 import { LikedVideos } from "./pages/index";
 import { WatchLater } from "./pages/index";
+import {History} from "./pages/index";
+import { SingleVideo } from "./pages/index";
```
Comment (pnchinmay) [inline src/App.js]: Barrel imports can be used here

File: src/components/video-card/video-card.css [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
     flex-wrap: wrap;
     gap: 3rem;
-    align-items: center;
-    justify-content: center;
 }
 
 .card{
-    cursor: pointer;
+    cursor: initial;
     position: unset;
```
Comment (pnchinmay) [inline src/components/video-card/video-card.css]: 2nd position property is overriding this one

File: src/components/video-card/video-card.js [hunk 3]
```diff
@@ -1,33 +1,47 @@
...
+import { useNavigate, useLocation } from "react-router-dom";
 import { useAuthContext } from "../../contexts/auth-context";
 import { useVideoContext } from "../../contexts/video-context";
-import { addToLikedVideos } from "../../utils/handleLikedVideos";
-import { removeFromLikedVideos } from "../../utils/handleLikedVideos";
-import { saveToWatchLater } from "../../utils/handle-watch-later";
-import { removeFromWatchLater } from "../../utils/handle-watch-later";
+import { addToLikedVideos, removeFromLikedVideos } from "../../utils/handleLikedVideos";
+import { saveToWatchLater, removeFromWatchLater } from "../../utils/handle-watch-later";
+import {addToHistory, deleteFromHistory, clearAllHistory} from "../../utils/handle-history"
```
Comment (pnchinmay) [inline src/components/video-card/video-card.js]: Barrel imports can be used here as well

**Review by pnchinmay — APPROVED** [review]
Suggested some changes

**Pull request merged**